apps/web/src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -105,15 +105,16 @@
         <dashboard-widget-edit-modal v-model:visible="state.visibleEditModal"
                                      :widget-config-id="props.widgetConfigId"
                                      :widget-key="props.widgetKey"
-                                     @refresh="handleRefresh"
+                                     @refresh="emit('refresh')"
```

Comment:
nitpick:
```suggestion
                                     @refresh="handleRefresh"
```


```
const handleRefresh = () => {
    emit('refresh');
}
```

This is our new convention!